adoc/i-Introduction/SA.adoc

```diff
@@ -1,40 +1,42 @@
-
 == Introduction
 
 *SUSE’s Vision is to Simplify, Modernize, Accelerate the business of our customers.*
 
-Maintaining a competitive advantage often depends on how quickly you can deliver new services. SAP applications are designed to help your business analyze data to anticipate new requirements, and rapidly deliver new products and services. This gives you the ability to keep your existing customers happy while attracting new business.
+Maintaining a competitive advantage often depends on how quickly new services can be delivered into a business. SAP applications are designed to help analyze data to anticipate new requirements, and rapidly deliver new products and services. 
 
-When we first released SUSE Linux Enterprise Server for SAP Applications, it already included automated installation features for the SAP software stack. Over the last 10 years, our SAP LinuxLab and development engineers have introduced several additional features to automate routine system administration.
+When SUSE first released SUSE Linux Enterprise Server for SAP Applications, it already included automated installation features for the SAP software stack. Over the last 10 years, the SUSE SAP LinuxLab and development engineers have introduced several additional features to automate routine system administration. Based on this experience, SUSE worked on reimagining the deployment wizard capability onto a modern framework.
 
 image::SLES4SAPBenefits.png[Benefits of SLES-for-SAP Applications,scaledwidth="80%"]
 
-*Simplify*::      the deployment of an SAP Landscape in Azure for dev, test, and production.
+*Simplify*::      the deployment of an SAP Landscape in The Public Cloud for dev, test, and production.
```

Comment:
Will change "public cloud"  back to Azure but use it as variable {cloud}  so it features the CSP

Reply:
I will leave you to change.